lib-injection/Dockerfile

```diff
@@ -29,6 +29,7 @@ ARG UID=10000
 RUN addgroup -g 10000 -S datadog && \
     adduser -u ${UID} -S datadog -G datadog
 RUN chown -R datadog:datadog /datadog-init/ddtrace_pkgs
+RUN chmod -R a+rwx /datadog-init/ddtrace_pkgs
```

Comment:
a+rwx is very permissive but I think this is okay.

Reply:
do any of the files have the execute permission (prior to us setting it)? If not, then I think we should avoid setting it